reports-backend/src/main/java/org/jboss/da/listings/api/model/GA.java

```diff
@@ -0,0 +1,33 @@
+package org.jboss.da.listings.api.model;
+
+import javax.persistence.Entity;
+
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+/**
+ * 
+ * @author Jozef Mrazek <jmrazek@redhat.com>
+ *
+ */
+@RequiredArgsConstructor
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@ToString
+@Entity
+public class GA extends GenericEntity {
+
+    @Setter
+    @Getter
+    @NonNull
+    private String groupId;
+
+    @Setter
+    @Getter
+    @NonNull
+    private String artifactId;
+}
```

Comment:
I'm not 100% sure, but probably there should be:

@OneToMany(mappedBy=..., fetchType=...)
private Set<BlackArtifact> blackArtifacts;

@OneToMany(mappedBy=..., fetchType=...)
private Set<WhiteArtifact> whiteArtifacts;


@janinko @thescouser89 What do you think?